auth-web/src/components/auth/MemberDataTable.vue

```diff
@@ -214,8 +215,8 @@ export default class MemberDataTable extends Vue {
         }
         return false
       case MembershipType.Coordinator:
-        // Admins can change roles of their own
-        return this.isOwnMembership(memberBeingChanged)
+        // Admins can change roles of their own and users as well
+        return this.isOwnMembership(memberBeingChanged) || (memberBeingChanged.membershipTypeCode !== MembershipType.Admin)
```

Comment:
This check may fail. since its an 'or', it will allow the admin to change the other Coordinator as well, right?
also, the comment need to be updated to new role

Reply:
plz check now..I fixed it